Utils/hydro_enforce_inout_solvability.cpp

```diff
@@ -59,6 +63,8 @@ void set_inout_masks(
             for (MFIter mfi(*vel_mf, TilingIfNotGPU()); mfi.isValid(); ++mfi) {
 
                 Box box = mfi.tilebox();
```

Comment:
Ideally, I wouldn't use a level_mask array, I would just check if this box is covered by finer cells and skip it entirely. However, I don't know if that is possible, and I'm familiar with makeFineMask from the AMR-Wind context, so that's what I used.